osu.Game/Scoring/RankingTier.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Scoring
+{
+    public enum RankingTier
```

Comment:
I dunno about this going in the scoring namespace.

Reply:
Couldn't come up with anything better so thought that since it's related to scores/ranks/levels it kind of fits in scoring

Reply:
Really depends on if/when this is ever used in the future. We go through so many colour schemes I wouldn't be surprised if it's replace ten times over before consumed anywhere.

Probably okay for now, but please add summary xmldoc mentioning that this is explicitly used for levels/colours and nothing else. With a name like `RankingTier` in a namespace like `Scoring` it's just adding further confusion to all the existing named things we have.

In fact, how about renaming to `LevelTier` or something for now? And moving to the user profile namespace (or graphics if that is too specific)?

Reply:
If we were to rename it into `LevelTier` we might as well remove the whole enum completely and do level->colour conversion straight in the `LevelBadge`. The place that I think I'm sure it'll be used is rank numbers in profile overlay

Reply:
Yeah I dunno. I just want to avoid another class that is used in one or two places which is doing nothing but adding confusion to all the existing classes with `Rank` in their name. But I guess there's not that many so let's just go with it until someone complains.